network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/exclusion/ui/NetpAppExclusionListViewModel.kt

```diff
@@ -173,24 +173,34 @@ class NetpAppExclusionListViewModel @Inject constructor(
     }
 
     private fun getAppsForExclusionList(): Flow<List<NetpExclusionListApp>> {
```

Comment:
There's a lot of logic now in this function. Can you add a comment here that explains inputs and expected output?